pkg/registry/client.go

```diff
@@ -685,19 +685,9 @@ func (c *Client) Push(data []byte, ref string, options ...PushOption) (*PushResu
 	})
 
 	ociAnnotations := generateOCIAnnotations(meta, operation.creationTime)
```

Comment:
I think the `parsedRef, err := newReference(ref)` above should be removed

Reply:
Not sure I get it. The `parsedRef` is used here ? 🤔

Reply:
Line 611. It won't let me comment up there

Reply:
Is that what you had in mind? https://github.com/helm/helm/pull/30894/commits/2c01f66d8b7d455ba9962018517d2bc87da15d2b

Reply:
Ah, yeh, that works